packages/@lightningjs/ui-components/src/mixins/withLongPress/index.js

```diff
@@ -0,0 +1,77 @@
+﻿/**
+ * Copyright 2023 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export default function withLongPress(Base) {
+  return class extends Base {
+    static get name() {
+      return Base.name;
+    }
+    set threshold(value) {
+      this._threshold = value;
+    }
+
+    set executeOnce(val) {
+      this._executeOnce = val;
+    }
```

Comment:
From our in-person discussion, let's change this to something like `continuousExecution` with a default value of `false` to make it clearer that the mixin allows for the signal to be fired repeatedly during a long press.